tests/src/sanity.rs

```diff
@@ -606,7 +606,6 @@ mod tests {
         }
     }
 
-    #[ignore] // TODO: create integration test instead.
```

Comment:
Why'd you remove it?

Reply:
want to see that it works since we don't have much coverage here, will re-add once it passes.